cluster/addons.go

```diff
@@ -39,14 +45,113 @@ func (c *Cluster) deployUserAddOns(ctx context.Context) error {
 	log.Infof(ctx, "[addons] Setting up user addons..")
 	if c.Addons == "" {
 		log.Infof(ctx, "[addons] No user addons configured..")
+		return c.deployAddonsInclude(ctx)
```

Comment:
the log message here might be confusing since you are still running user addon includes